### PR TITLE
Bug 1068880 - imporeved accessibility around camera settings UI.

### PR DESCRIPTION
**Pull request by yzen**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[yzen Yura Zenevich (yzen)](https://github.com/yzen) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=66b037f5ac00)

File: apps/camera/js/controllers/controls.js [hunk 1]
```diff
@@ -57,6 +57,10 @@ ControlsController.prototype.bindEvents = function() {
   this.app.on('timer:cleared', this.onTimerStopped);
   this.app.on('timer:ended', this.onTimerStopped);
 
+  // Settings
+  this.app.on('settings:opened', this.onSettingsOpened);
```
Comment (justindarc) [inline apps/camera/js/controllers/controls.js]: Couldn't you just do:

`this.view.setter('ariaHidden', true/false);`

like you do for the indicators?

Comment (yzen) [inline apps/camera/js/controllers/controls.js]: @justindarc , unfortunately, set is overridden by the controllers view and filters for data attributes.

File: apps/camera/style/settings.css [hunk 2]
```diff
@@ -11,12 +11,15 @@
   background: rgba(0,0,0,0.85);
   overflow: hidden;
   opacity: 1;
-  transition: opacity 150ms linear;
-  will-change: opacity;
+  transition: opacity 150ms linear, visibility 0s;
+  visibility: visible;
+  will-change: opacity, visibility;
```
Comment (justindarc) [inline apps/camera/style/settings.css]: I don't think `visibility` needs to be called out explicitly for the `will-change` property. The only reason this is here is to promote the settings panel to its own layer in the layer tree.

File: apps/camera/js/controllers/controls.js (hunk 1 above)
Comment (justindarc) [inline apps/camera/js/controllers/controls.js]: Ah, yeah you're right.

**Comment by try-server-hook** [comment]
[yzen Yura Zenevich (yzen)](https://github.com/yzen) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=15361b4c7f83)

**Comment by try-server-hook** [comment]
[yzen Yura Zenevich (yzen)](https://github.com/yzen) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=65d940396f52)

**Comment by try-server-hook** [comment]
[yzen Yura Zenevich (yzen)](https://github.com/yzen) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=7cd0e45739ab)

**Pull request merged**